src/mesh/parallel_mesh.C

```diff
@@ -1349,14 +1349,14 @@ void DistributedMesh::delete_remote_elements()
 {
 #ifdef DEBUG
   // Make sure our neighbor links are all fine
-  MeshTools::libmesh_assert_valid_neighbors(*this);
+  //MeshTools::libmesh_assert_valid_neighbors(*this);
```

Comment:
Sorry guys: I didn't mean to leave these changes in here.  Those just slipped through.  I'll revert those and force-push this.

BTW: As you can see with my newest commit... I'm still working on this a bit.  One of my problems still won't run using this capability.  Still hunting down why.